src/mmw/js/src/data_catalog/templates/resultDetailsCuahsiTableRowVariableCol.html

```diff
@@ -1,4 +1,7 @@
 {{ concept_keyword }}
+{% if data_type and data_type != 'Unknown' and data_type != 'Instantaneous' %}
```

Comment:
Since this logic's repeated 2x would it make sense to move it up into the view as a `templateHelper`?

Reply:
They are in separate views, so we'd have to put it in two `templateHelper`s. To effectively de-duplicate, we could move it to within the model itself, but this seems more like a presentation concern than a data concern, so preferred to keep it in the markup.